cmd/policy-assistant/pkg/cli/analyze.go

```diff
@@ -296,59 +308,114 @@ func shouldIncludeANPandBANP(client *kubernetes.Clientset) (bool, bool) {
 	return includeANP, includeBANP
 }
 
-func VerdictWalkthrough(policies *matcher.Policy) {
+func VerdictWalkthrough(policies *matcher.Policy, sourceWorkloadTraffic string, destinationWorkloadTraffic string, port int, protocol string, trafficPath string) {
+	var sourceWorkloadInfo matcher.TrafficPeer
+	var destinationWorkloadInfo matcher.TrafficPeer
+	var allTraffic []*matcher.Traffic
+
+	if trafficPath != "" && (sourceWorkloadTraffic != "" || destinationWorkloadTraffic != "" || port != 0 || protocol != "") {
+		logrus.Fatalf("%+v", errors.Errorf("If using traffic path, you can't input traffic via CLI and viceversa"))
```

Comment:
Thinking about how a user will know what parameters to use for this mode. Think you solved the first problem of specifying both traffic path and other args.

Could we also add a check up here that if traffic path is not set, then all the other args must be set.

I would also word this existing check and the new check to explicitly write out the arg names because there's currently no place where the user would learn which args need to be set for "walkthrough" mode. Something like: "for this mode, must either set --traffic-path or set all of --source-workload --dst-workload..."